Payment system

ktamas77 opened the pull request:
#### Description

Proportional Distribution Payment System with ILP
Note: Database for Development is persistent

#### Issues

List of issues that need to be closed for closing this milepost:

- [ ] #1 - Finish Contribution Page
- [ ] #2 - Finish Payment to Group Page
- [ ] #3 - Add Payment Options for Prifles
- [ ] #4 - Add ILP

ktamas77 committed:
Add Group button fixed

ktamas77 committed:
Merge branch 'master' of https://github.com/okTurtles/group-income-simple into payment-system

ktamas77 committed:
Merge

ktamas77 committed:
Pay Group Members screen

ktamas77 committed:
Proportional Mincome Distributon added to Pay to Groups

taoeffect commented:
NOTE: package-lock.json is modified in this PR and it's not clear why. See if there's a way to prevent that via some `npm install` command switches or whatever

taoeffect committed:
Pubsub sync + SBL!

+ Pubsub sync. Closes #298
+ Basic SBL implementation + usage! :D
+ Improvements to SignUp.vue implementation

vijayee committed:
Merge pull request #315 from taoeffect/pubsub-sync

Pubsub sync + SBL!

taoeffect committed:
Renamed SBL to SBP

Selector-based Programming/Paradigm

vijayee committed:
Merge pull request #318 from taoeffect/pubsub-sync

Renamed SBL to SBP

hubudibu committed:
fix input icons on signup and login views

taoeffect committed:
Merge pull request #322 from hubudibu/fix-bulma-icons

Fix input icons on signup and login views

hubudibu committed:
update package-lock: order changes only

hubudibu committed:
group creation button got its text back

hubudibu committed:
let's be consistent with the v-ifs

and not just visually hide the options one does not have authorization for

taoeffect committed:
Merge pull request #323 from hubudibu/package-lock-sort

Update package-lock order

taoeffect committed:
Merge pull request #324 from hubudibu/fix/create-group-link

Fix group creation link in navbar

ktamas77 committed:
Merge branch 'payment-system' of https://github.com/okTurtles/group-income-simple into payment-system

ktamas77 commented:
cleaning up this request
